services/web-app/pages/about-carbon/faq/index.mdx

```diff
@@ -18,110 +20,100 @@ description:
+its foundation, the system consists of working code, design tools and resources, human interface
+guidelines, and a vibrant community of contributors.
 
-## Contributing
+To learn more about the Carbon ecosystem, check out
+[how Carbon works](/about-carbon/how-carbon-works).
 
-### How can I contribute and/or propose new components or ideas?
+### What is Carbon Next?
 
-Check out our [contribution overview](/contributing/overview) to learn all about the different ways
-to contribute. Thanks for the help!
+Carbon Next is a project to create a new version of the Carbon website, as well as new
+infrastructure to support indexing and maintaining assets. This will eventually replace the existing
+Carbon and PAL websites.
 
-### I see a bug. How do I report it?
+Check out the [platform roadmap](/about-carbon/platform-roadmap) to learn more about planned
+releases.
 
-First, make sure the problem is reproducible. Once confirmed, open an issue in the appropriate
-GitHub repo from the following list. We will address the bug as soon as we can. If you have a fix
-for the bug, please feel free to submit a PR for it.
+### Why is the Carbon website changing?
 
-For larger changes, please see our [contribution overview](/contributing/overview).
+In the past, open and inner source assets were scattered across multiple websites. The new site
+provides a unified discovery experience, which helps designers and developers find and access
+components, patterns, functions and templates across all IBM teams. This will help everyone find
+assets that comply with platform requirements, are convenient to implement, and are consistent with
+design patterns across the company.
 
-#### Design issue repos
-
-- [Design kit](https://github.com/carbon-design-system/carbon-design-kit/issues/new)
-- [Icons](https://github.com/carbon-design-system/carbon-icons/issues/new)
-- [Website](https://github.com/carbon-design-system/carbon-website/issues/new/choose)
-- [Components](https://github.com/carbon-design-system/carbon/issues/new/choose)
-- [Elements](https://github.com/carbon-design-system/carbon/issues/new/choose)
-- [Charts](https://github.com/carbon-design-system/carbon-charts/issues/new)
-- [Everything else](https://github.com/carbon-design-system/issue-tracking/issues/new)
+## Creating with Carbon
 
-#### Code repos
+### Where do I start if I’m a developer?
 
-- [@carbon/react](https://github.com/carbon-design-system/carbon/issues/new/choose)
-- [carbon-components-angular](https://github.com/IBM/carbon-components-angular/issues/new)
-- [carbon-components-vanilla](https://github.com/carbon-design-system/carbon/issues/new/choose)
-- [carbon-components-vue](https://github.com/carbon-design-system/carbon-components-vue/issues/new)
-- [carbon-charts](https://github.com/carbon-design-system/carbon-charts/issues/new)
+Head [here](/developing) to learn about libraries and access everything you need to get up and
```

Comment:
Do we want to re-word this so the links are more descriptive vs. just saying "here" ?

Reply:
Good idea. Just committed changes.